app/assets/stylesheets/application.css.scss

```diff
@@ -119,3 +121,7 @@ a:hover {
 #footer {
     background:#000;
 }
+
+#username-link a, a:link, a:visited {
```

Comment:
Avoid using id selectors<br>Each selector in a comma sequence should be on its own line